dijkstra/dijkstra.c

```diff
@@ -16,7 +16,7 @@
 int minimum_distance(const int min_distances[], const bool shortest_paths[]) {
 	int i;
 	int minimum = INFINITY, index;
-	for (int i = 0; i < VERTICES; ++i) {
+	for (i = 0; i < VERTICES; ++i) {
```

Comment:
@Monal5031: This implementation had a compile time error, yet you submitted it. This means that you didn't find it necessary to compile a code before submitting a patch, which is bad. Please work in it.